app/uk/gov/hmrc/helptosavefrontend/forms/BankDetailsValidation.scala

```diff
@@ -80,18 +84,14 @@ class BankDetailsValidation @Inject() (configuration: FrontendAppConfig) {
     override def bind(key: String, data: Map[String, String]): Either[Seq[FormError], Option[String]] = {
       val validation: ValidOrErrorStrings[Option[String]] =
         data.get(key)
-          .map(_.cleanupSpecialCharacters.removeAllSpaces)
           .filter(_.nonEmpty)
-          .fold[ValidOrErrorStrings[Option[String]]](Valid(None)){ s ⇒
-            if (s.length < rollNumberMinLength) {
-              invalid(ErrorMessages.rollNumberTooShort)
-            } else if (s.length > rollNumberMaxLength) {
-              invalid(ErrorMessages.rollNumberTooLong)
-            } else {
+          .fold[ValidOrErrorStrings[Option[String]]](Valid(None)) { s ⇒
```

Comment:
don't we need to do the same validation in the `help-to-save-api` as well ?

Reply:
the validation in the proxy should be enough for the API journey - we have it in the FE since we can stop people in the journey before attempting to create an account if their bank details are wrong

Reply:
i checked in the  https://regex101.com/ and i have a feeling that `^([0-9a-zA-Z\/.-]{4, 18})$` is the correct one to use 
but the spec has `\\/\\`